src/main/java/com/minecolonies/inventory/InventoryCitizen.java

```diff
@@ -258,9 +258,9 @@ public boolean addItemStackToInventory(@Nullable final ItemStack itemStackIn)
                 {
                     crashreportcategory.addCrashSection("Item name", itemStackIn.getDisplayName());
                 }
-                catch (Throwable throwable)
+                catch (RuntimeException e)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Catch a list of specific exception subtypes instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2221)